packages/wallet/ui/src/components/Offer.jsx

```diff
@@ -194,17 +196,15 @@ const OfferWithoutContext = ({
     </div>
   );
 
+  const isOfferCompleted =
+    status === 'accept' ||
+    status === 'decline' ||
+    status === 'complete' ||
+    status === 'rejected' ||
+    status === 'cancel';
+
```

Comment:
```suggestion
  const isOfferCompleted = ['accept', 'decline', 'complete', 'rejected', 'cancel'].includes(status);
```

Reply:
@samsiegart @michaelfig 
Where does this status string come from? Where are its possible values defined?

Reply:
I'm not sure if we have any good documentation on that. The implementation is in https://github.com/Agoric/agoric-sdk/blob/master/packages/wallet/api/src/lib-wallet.js (search for "status:")